src/screens/types-of-suffering/dialog-item.js

```diff
@@ -18,17 +18,24 @@ export function DialogItem({ id, title, description }) {
       </RadixDialog.Trigger>
       <RadixDialog.Portal>
         <RadixDialog.Overlay className="fixed inset-0 z-10 h-screen w-screen bg-[#1a3439]/50" />
-        <RadixDialog.Content className="fixed inset-0 z-10 flex h-screen w-screen flex-col">
-          <div className="relative m-auto grid w-full max-w-4xl grid-cols-2 place-items-center rounded-2xl border-4 border-white bg-white bg-sand p-10">
+        <RadixDialog.Content className="fixed bottom-0 z-10 max-h-screen w-full max-w-4xl overflow-y-auto md:left-1/2 md:top-1/2 md:-translate-y-1/2 md:-translate-x-1/2">
```

Comment:
The usage of top/left 50% with negative translates has some issues when the viewport is smaller than the actual content, if I'm not mistaken. Using `flex` and `m-auto` on the child should fix this issue.